DuckDuckGo/TabViewController.swift

```diff
@@ -1852,6 +1853,34 @@ extension TabViewController: WKNavigationDelegate {
     @objc private func dismissLoginDetails() {
         dismiss(animated: true)
     }
+
+    private func registerForAutofillNotifications() {
```

Comment:
As this class is already a bit of a beast, do you think it'd be possible to extract the breakage reporting into another type?